metrics.md

```diff
@@ -295,11 +295,52 @@ Uncertainty is the variance of the event indicator $$ o(t) $$. Low values of UNC
 {: .anchor }
 The BSS is based on the BS and measures the performance of a probability forecast relative to a reference forecast:
 
-$$ BSS = 1 - \frac{\text{BS}_f}{\text{BS}_{\text{ref}}} $$
+$$ \text{BSS} = 1 - \frac{\text{BS}_f}{\text{BS}_{\text{ref}}} $$
 
 where $$ \text{BS}_f $$ is the BS of the forecast of interest, and $$ \text{BS}_{\text{ref}} $$ is the BS of the reference forecast. BSS greater than zero indicates the forecast performed better than the reference and vice versa for BSS less than zero, while BSS equal to zero indicates the forecast is no better (or worse) than the reference.
 
 
+### Quantile Score (QS) {#qs}
+{: .anchor }
+QS measures the accuracy of quantile forecasts, in which the forecast predicts the variable value corresponding to a constant probability ([Koenker78](#ref-koenker78), [Wilks11](#ref-wilks11)). QS is similar to BS, but measures accuracy in terms of the variable value (e.g. MW) and is defined as:
```

Comment:
I believe quantile score is only in the 4th edition of Wilks. Easiest thing is probably to add an new reference for that and leave the Wilks 2011 reference alone. Or we can double check that all of the existing references to Wilks remain good in the new edition and replace it.

Reply:
If it's fine with you, let's just start with adding the 4th edition as a separate reference.